tests/handler_test.py

```diff
@@ -128,6 +131,19 @@ def _check_batch_details(ips, details, token):
             assert "domains" in d
 
 
+def _check_iterative_batch_details(details, token):
```

Comment:
Can we just use `_check_batch_details`? Just pass in a single IP array containing the returned detail's IP

Reply:
No, actually `getBatchDetails` returns `dict` as we can simply loop through the entries and check the details. On the other hand `getBatchDetailsIter` is returning generator object and we yield them iteratively. I tried storing it in separate variable first before yield, but it gets override by new value so it arrives. Also we have to do some quirks to get the detail value as it is not subscriptable.